Source/Core/VideoCommon/TextureDecoder_x64.cpp

```diff
@@ -1486,6 +1487,56 @@ void _TexDecoder_DecodeImpl(u32* dst, const u8* src, int width, int height, Text
     TexDecoder_DecodeImpl_CMPR(dst, src, width, height, texformat, tlut, tlutfmt, Wsteps4, Wsteps8);
     break;
 
+  case TextureFormat::XFB:
+  {
+    for (int y = 0; y < height; y += 1)
+    {
+      for (int x = 0; x < width; x += 2)
+      {
+        size_t offset = static_cast<size_t>((y * width + x) * 2);
+
+        int inR = int(src[offset]);
+        int inG = int(src[offset + 1]);
+        int inB = int(src[offset + 2]);
+        int inA = int(src[offset + 3]);
+
+        // If our yuvyv value is equal to our 'out of range' key
+        // just return a transparent pixel
+        if (inR == 1 && inG == 254 && inB == 1 && inA == 254)
+        {
+          dst[y * width + x] = 0;
+          dst[y * width + x + 1] = 0;
+        }
+        else
+        {
+          // We do this one color sample (aka 2 RGB pixles) at a time
+          int Y1 = inR - 16;
+          int U = inG - 128;
+          int Y2 = inB - 16;
+          int V = inA - 128;
+
+          // We do the inverse BT.601 conversion for YCbCr to RGB
```

Comment:
Repeated comment?

Reply:
Probably, this was done in phire's original implementation, I left it as is.

Reply:
Done